src/app/modules/system-process/pages/peer-locations-map/peer-locations-map.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { take } from 'rxjs/operators';
+import { GeoCoordinate, GeoLocatorService } from '../../../core/services/geo-locator.service';
+declare let L: any;
+
+@Component({
+  selector: 'app-peer-locations-map',
+  templateUrl: './peer-locations-map.component.html'
+})
+export class PeerLocationsMapComponent implements OnInit {
+
+  constructor(
+    private geoLocationService: GeoLocatorService
+  ) { }
+
+  ngOnInit(): void {
+    const locationMap = L.map('peer-locations-map').setView([48, 32], 2.6);
+    const mapIcon: any = L.icon({
```

Comment:
Let's avoid `any` at all costs, in fact, we should add a linter rule that fails the project if we use any unless there is an exceptional case

Reply:
wow that was a lot of work for removing 3 any's .. added a leaflet file with basic typescript support for all the things we use.